packages/fxa-auth-server/lib/routes/password.ts

```diff
@@ -697,8 +696,13 @@ module.exports = function (
         statsd.increment('otp.passwordForgot.attempt');
 
         const { email, code } = request.payload;
+
+        // Typical 15 minute window limit
         await customs.check(request, email, 'passwordForgotVerifyOtp');
 
+        // Daily limit, will be checked if and only if the default limit above passes
+        await customs.check(request, email, 'passwordForgotVerifyOtpPerDay');
```

Comment:
Also not a defined action in customs.

Reply:
I'm going to guard this with a check about if v2 is enabled.